.github/workflows/backend_integration_tests_pr.yml

```diff
@@ -12,7 +12,7 @@ on:
       - edited
 
 concurrency:
-  group: backend-e2e-${{ github.event.number }}
+  group: backend-e2e-${{ github.event.number || github.event.pull_request.number}}
```

Comment:
extra: I saw that sometimes these tests were getting canceled. I suspect is due to `github.event.number` being null on some cases